src/plugins/share/share.js

```diff
@@ -94,7 +94,7 @@ var componentName = "wb-share",
 				url: "https://www.tumblr.com/share/link?url={u}&amp;name={t}&amp;description={d}"
 			},
 			twitter: {
-				name: "Twitter",
+				name: "X",
 				url: "https://twitter.com/intent/tweet?text={t}&url={u}"
 			},
```

Comment:
Add support for web author that want to filter by specifying "X" instead of "twitter"

```suggestion
			},
			x: {
				name: "X",
				url: "https://twitter.com/intent/tweet?text={t}&url={u}"
			},
```

And you can add a working example like:
```html

	<section>
		<h3>With "X" instead of "twitter"</h3>

		<div class="wb-share" data-wb-share='{"filter": ["email", "facebook", "linkedin", "x"], "pnlId": "pnl5a"}'></div>

		<details>
			<summary>View code</summary>
			<pre class="wb-prettify"><code>&lt;div class="wb-share" data-wb-share='{"filter": ["email", "facebook", "linkedin", "x"], "pnlId": "pnl5a"}'&gt;&lt;/div&gt;</code></pre>
		</details>
	</section>
</section>
```

And it might require you to add a new image, named "x.png" to our sprite images.